.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+# See http://pre-commit.com/
```

Comment:
We're adding pre-commit as well so we ensure we commit formatted code before we open a PR.